api/src/extensions/lib/sandbox/register/operation.ts

```diff
@@ -18,7 +18,11 @@ export function registerOperationGenerator() {
 
 		const idCopied = id.copySync();
 
-		const handler: OperationHandler = async (data) => callReference(cb, [data]);
+		const handler: OperationHandler = async (payload) => {
+			const response = await callReference(cb, [payload]);
```

Comment:
Just a small nitpick: Should we call the parameter of the handler `options` to be consistent with what it is called on the Type?

Reply:
Sounds good 👌